common/app/awswrappers/kinesisfirehose.scala

```diff
@@ -1,13 +0,0 @@
-package awswrappers
-
-import com.amazonaws.services.kinesisfirehose.AmazonKinesisFirehoseAsync
-import com.amazonaws.services.kinesisfirehose.model.{PutRecordRequest, PutRecordResult}
-
-import scala.concurrent.Future
-
-object kinesisfirehose {
-  implicit class RichKinesisFirehoseAsyncClient(client: AmazonKinesisFirehoseAsync) {
```

Comment:
This was introduced in:

* https://github.com/guardian/frontend/pull/19406

..but it is unused since this PR:

* https://github.com/guardian/frontend/pull/27755